### ISSUE TITLE
Allow User to Search  book by Author name.

### ISSUE THREAD

**Issue by Gaurav-64-PerennialSystem** [Gaurav-64-PerennialSystem/BookSharingApplication#3]
Description
A user should be able to search book by authorname.

Search book by authorname.
 1.search by authorname-mandatory.

Validations
1.Name should contains only alphabets not  number or other special characters.

Acceptance Criteria
1.List of search book should be displayed.
2.Details of books-book name ,authorname should be displayed.

**Comment by udayraj-perennial** [comment]
Can we search with partial author name? Wil the space in the name be allowed?

**Comment by Gaurav-64-PerennialSystem** [comment]
Yes we can search by partial  author name by using regular expression also space can be allowed by the regular expression.
